react/__tests__/components/ProductPrice.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ProductPrice from './../../ProductPrice'
+import { render } from 'enzyme'
+
+describe("<ProductPrice />", () => {
+  let renderComponent = null
```

Comment:
Instead of using a global variable, why not having a function that generates and returns the component to be tested